packages/application-shell/src/utils/gtm.ts

```diff
@@ -114,7 +114,7 @@ export const trackTiming = ({
 }: {
   category: string;
   variable: string;
-  value: string;
+  value: unknown;
```

Comment:
Can you share why we need this to be unknown. Fail to follow.

Reply:
In theory it's a string or a number. At this point the exact type does not matter much, but yeah let's be more specific.